### PR TITLE
Add blog section

### PR DESCRIPTION
**Pull request by fpiwowarczyk**
Introduce new functionality to create blog posts on OpenSLO site.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
 The flow is the same as for regular contributions, review will be made in a PR.
 
 For something more complicated or when you are in doubt please create an issue first to gather feedback.
+
+## Creating blog post 
+
+Please follow while creating new post on our blog:
```
Comment (programmer04) [inline README.md:20]:
```suggestion
Please follow while creating a new post on our blog:
```

File: README.md [hunk 2]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
...
+
+1. Create file with name of post
```
Comment (programmer04) [inline README.md:22]:
```suggestion
1. Create a file with the name of the post 
```

File: README.md [hunk 3]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
...
+## Creating blog post 
+
+Please follow while creating new post on our blog:
+
+1. Create file with name of post 
+```sh
+hugo new posts/{post-name}.md
+```
+2. In path /content/posts you'll find your newly created file 
+3. Please add in front matter variable called `cover` with value pointing to cover image of your post. You can link image or use one which should be placed in /assets directory.
```
Comment (programmer04) [inline README.md:27]:
```suggestion
3. Please add in the front matter variable called `cover` with a value pointing to the cover image of your post. You can link an image or use one which should be placed in `/assets` directory. 
```

File: README.md [hunk 4]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
...
+
+Please follow while creating new post on our blog:
+
+1. Create file with name of post 
+```sh
+hugo new posts/{post-name}.md
+```
+2. In path /content/posts you'll find your newly created file 
+3. Please add in front matter variable called `cover` with value pointing to cover image of your post. You can link image or use one which should be placed in /assets directory. 
+4. Fill post with content
```
Comment (programmer04) [inline README.md:28]:
```suggestion
4. Write your post
```

File: README.md [hunk 5]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
...
+5. If you finished your work change draft value to false
```
Comment (programmer04) [inline README.md:29]:
```suggestion
5. If you finished your work change `draft` value to `false`
```

File: content/posts/first_post.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Hello world!"
+date: 2022-08-18T13:00:42+02:00
+draft: false
+cover: /images/openslo_logo.svg
+---
+First blog post on OpenSLO blog section. Here we will be posted all news and 
+interesting initiatives. If you are interested in writing your own blog post
+create pull request on our [Github repository](https://github.com/OpenSLO/openslo.github.io).
```
Comment (programmer04) [inline content/posts/first_post.md:9]:
```suggestion
create a pull request on our [GitHub repository](https://github.com/OpenSLO/openslo.github.io).
```

File: README.md [hunk 7]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
...
+
+Example front matter
```
Comment (programmer04) [inline README.md:31]:
```suggestion
Example front matter for the post
```

**Review by programmer04 — APPROVED** [review]
Some nit picks, good job!

File: README.md [hunk 8]
```diff
@@ -14,3 +14,25 @@ This website contains a blog section, everybody interested in SLO, SRE, OpenSLO,
 The flow is the same as for regular contributions, review will be made in a PR.
 
 For something more complicated or when you are in doubt please create an issue first to gather feedback.
+
+## Creating blog post
```
Comment (programmer04) [inline README.md:18]:
```suggestion
## Creating a blog post 
```

**Comment by fpiwowarczyk** [comment]
> Some nit picks, good job!

Thanks, your suggestions commited ;)

**Pull request merged**